ui/src/assets/chat_page.scss

```diff
@@ -0,0 +1,110 @@
+/* Make the main container a flex column that fills the available height */
+.chat-container {
+  display: flex;
+  flex-direction: column;
+  height: 95vh;
+  /* 95 viewport height */
+  width: 100%;
```

Comment:
Flex = block which fills parent horizontally by default so I think this is redundant